apps/src/blockly/addons/cdoXml.js

```diff
@@ -35,8 +35,39 @@ export default function initializeBlocklyXml(blocklyWrapper) {
   };
 
   blocklyWrapper.Xml.domToBlockSpace = function(blockSpace, xml) {
+    // To position the blocks, we first render them all to the Block Space
+    //  and parse any X or Y coordinates set in the XML. Then, we store
+    //  the rendered blocks and the coordinates in an array so that we can
+    //  position them in two passes.
+    //  In the first pass, we position the visible blocks. In the second
+    //  pass, we position the invisible blocks. We do this so that
+    //  invisible blocks don't cause the visible blocks to flow
+    //  differently, which could leave gaps between the visible blocks.
+    const blocks = [];
+    /**
+     * NodeList.forEach() is not supported on IE. Use Array.prototype.forEach.call() as a workaround.
```

Comment:
Nit: I know you didn't write this code, but since we don't support IE any more, I wonder if you could just `xml.childNodes.forEach(xmlChild => {})` now?

Reply:
Good call! Fixed this one and another in cdoTrashcan while I was at it.